src/modules/editingTask.js

```diff
@@ -0,0 +1,46 @@
+import modifyTask from './modifyTask.js';
+
+function finishEditing(event) {
+  const selectedElementClass = `.${event.currentTarget.parentNode.parentNode.classList[0]}`;
+  // `.${event.path[2].classList[0]}`
+  // console.log(selectedElementClass);
+  // console.log(event.currentTarget.parentNode.parentNode.classList[0]);
+  const elementLi = document.querySelector(selectedElementClass);
+  const elementInput = event.currentTarget;
+  elementLi.classList.remove('editing-background');
+  elementInput.classList.remove('editing-background');
+  elementInput.addEventListener('input', modifyTask(elementInput.value, selectedElementClass));
+  const icon = document.querySelector(`${selectedElementClass} > img`);
+  icon.setAttribute('src', './assets/images/dot.png');
+}
+
+function editingTask(event) {
+  // console.log(event.currentTarget.classList[0]);
+  const elementLi = event.currentTarget;
+  const elementInput = event.currentTarget.firstElementChild.lastElementChild;
+  elementInput.addEventListener('blur', finishEditing);
+  elementLi.classList.add('editing-background');
+  elementInput.classList.add('editing-background');
+  const icon = document.querySelector('.editing-background > img');
+  icon.setAttribute('src', './assets/images/trash-can.png');
+  const p = event.currentTarget;
+  icon.addEventListener('click', () => {
+    const list = JSON.parse(localStorage.getItem('Tasks'));
+    const newlist = [];
+    list.forEach((element) => {
```

Comment:
- [ ] I am getting the following errors when I  try to add a new item. Kindly fix this
![Screenshot from 2022-04-30 18-13-06](https://user-images.githubusercontent.com/62705947/166113553-6baae38a-f5c0-4c74-a09b-abb5bc5e87b4.png)